.travis.yml

```diff
@@ -2,15 +2,16 @@ language: julia
 os:
   - linux
 julia:
-  - 0.6
+  - 0.7
 env:
   - PYTHON=""
 notifications:
   email: false
 sudo: false
 script:
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia -e 'Pkg.clone(pwd()); Pkg.build("TimeSeries"); Pkg.test("TimeSeries", coverage=true)'
+  - julia -e 'Pkg.clone(pwd()); Pkg.build("TimeSeries")'
+  - julia -e 'Pkg.test("TimeSeries", coverage=true)'
```

Comment:
For compatibility with 0.7/1.0, you should just remove or comment out the `script` block entirely. That allows Travis to use its default script for Julia, which is set up to work with all existing versions.

Reply:
oh, okay.